tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,246 @@
+import axios from 'axios';
+import * as fs from 'fs';
+import * as https from 'https';
+import jwtDecode from 'jwt-decode';
+
+import test, {expect} from '../fixtures/fixtures';
+
+const timeoutFourSeconds = 4000;
+const timeoutOneSecond = 1000;
+const samlSettings = {
+  'credentials': {
+    'chrome': {
+      'login': 'morty',
+      'password': 'panic',
+      'email': 'msmith@samltest.id'
+    },
+    'firefox': {
+      'login': 'rick',
+      'password': 'psych',
+      'email': 'rsanchez@samltest.id'
+    },
+    'webkit': {
+      'login': 'sheldon',
+      'password': 'bazinga',
+      'email': 'scooper@samltest.id'
+    },
+  },
+  'idp_url': 'https://samltest.id/saml/idp'
+};
+let metadataId;
+let jwt;
+
+test.describe('SAML Login', () => {
+  test.use({storageState: 'storage.json'});
+  test.describe('SAML login via sso/id/login', () => {
+    test.afterAll(async ({baseUrl, samlCredentials}, testInfo) => {
+      console.log(`Finished ${testInfo.title} with status ${testInfo.status}. Cleaning up.`);
+      const response = await axios({
+        url: `${baseUrl}api/management/v1/useradm/users?email=${encodeURIComponent(samlSettings['credentials'][samlCredentials]['email'])}`,
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+          'Content-Type': 'application/json'
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        })
+      });
+
+      const responseStatus = await response.status;
+      if (responseStatus != 200 || response.data.length < 1) {
+        console.log(`${samlSettings['credentials'][samlCredentials]['email']} does not exist.`);
+      } else {
+        expect(response.status).toBe(200);
+        const userId = response.data[0]['id'];
+        const responseDelete = await axios({
+          url: `${baseUrl}api/management/v1/useradm/users/${userId}`,
+          method: 'DELETE',
+          headers: {
+            Authorization: `Bearer ${jwt}`,
+            'Content-Type': 'application/json'
+          },
+          httpsAgent: new https.Agent({
+            rejectUnauthorized: false
+          })
+        });
+        expect(response.status).toBe(200);
+        console.log(`removed user ${samlSettings['credentials'][samlCredentials]['email']}/${userId}.`);
```

Comment:
Nitpick:
```suggestion
        console.log(`removed user ${samlSettings.credentials[samlCredentials].email}/${userId}.`);
```
- smells too much like python and a lack of comfort otherwise 😝 (also the other occurrences)

Reply:
right! fixed everywhere, with login, password and idp_url.